tests/RabbitMQQueueTest.php

```diff
@@ -80,7 +82,9 @@ public function test_push()
             $this->config['queue_params']['passive'],
             $this->config['queue_params']['durable'],
             $this->config['queue_params']['exclusive'],
-            $this->config['queue_params']['auto_delete']
+            $this->config['queue_params']['auto_delete'],
+            false,
+            Mockery::any()
```

Comment:
Can you find a better way to test here?

If I add `new AMQPTable($this->config['queue_params']['arguments'])` it doesn't pass. Probably because objects are different. I could make it pass any other way but with `any()`...